be/src/storage/tablet.h

```diff
@@ -249,6 +249,25 @@ class Tablet : public BaseTablet {
         return _tablet_meta->set_enable_persistent_index(enable_persistent_index);
     }
 
+    bool get_enable_binlog() { return _tablet_meta->get_binlog_config()->version; }
```

Comment:
Why use `version` to represent enable or not?

Reply:
ok，it should be binlog_enable